### DISCUSSION TITLE
Content links for documentation

### DISCUSSION THREAD

**Discussion by sukhpreetssekhon** [Sunbird-RC/community#662] [General]
@Renuka-S @tejash-jl -  These are issues regarding documentation which are closed but there are no links to the documentation. Will be great if you could share the content links in these tickets so that the community can view them.

https://github.com/Sunbird-RC/community/issues/420
https://github.com/Sunbird-RC/community/issues/421
https://github.com/Sunbird-RC/community/issues/426
https://github.com/Sunbird-RC/community/issues/435
https://github.com/Sunbird-RC/community/issues/429

cc - @surendrasinghs @coolbung @chakshugautam

**Comment by Renuka-S** [comment]
@manukalia29 Could you please do the needful.

**Comment by manukalia29** [comment]
Hi @sukhpreetsamagra 

We have published the closed tasks on Sunbird RC's microsite.  Below are the links:
https://github.com/Sunbird-RC/community/issues/420   https://docs.sunbirdrc.dev/learn/electronic-registries/what-issues-will-electronic-registries-address
https://github.com/Sunbird-RC/community/issues/421 https://docs.sunbirdrc.dev/learn/readme-1/verbs
https://github.com/Sunbird-RC/community/issues/426 https://docs.sunbirdrc.dev/learn/verifiable-credentials/what-are-the-key-roles-in-verifiable-credentials
https://github.com/Sunbird-RC/community/issues/435 https://docs.sunbirdrc.dev/learn/readme-1/workflows
https://github.com/Sunbird-RC/community/issues/429 https://docs.sunbirdrc.dev/learn/verifiable-credentials/digital-credentials-vs-verifiable-credentials
